### PR TITLE
Config for darts-stub service

### PR DESCRIPTION
**Pull request by tonymort**
Config for darts-stub service

### CODE REVIEW

File: apps/darts-modernisation/darts-stub-services/stg.yaml [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+metadata:
+  name: darts-stub-services
+  namespace: darts-modernisation
+spec:
+  releaseName: darts-stub-services
+  values:
+    java:
+      ingressHost: darts-stub-services.staging.platform.hmcts.net
+      environment:
+        DARTS_API_URL: https://darts-stub-services.staging.platform.hmcts.net
```
Comment (davet1985) [inline apps/darts-modernisation/darts-stub-services/stg.yaml:12]: Is that correct?

File: apps/darts-modernisation/darts-stub-services/darts-stub-services.yaml [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+        namespace: flux-system
+      interval: 1m
+  values:
+    java:
+      image: sdshmctspublic.azurecr.io/darts/stub-services:prod-3d6481d-20230907150729
+      disableTraefikTls: true
+    function:
+      image: sdshmctspublic.azurecr.io/darts/stub-services:prod-3d6481d-20230907150729
+      environment:
+        ATS_MODE: true
```
Comment (davet1985) [inline apps/darts-modernisation/darts-stub-services/darts-stub-services.yaml:23]: Remove this part, you don't need it

File: apps/darts-modernisation/darts-stub-services/stg.yaml (hunk 1 above)
Comment (tonymort) [reply apps/darts-modernisation/darts-stub-services/stg.yaml]: fixed

File: apps/darts-modernisation/darts-stub-services/darts-stub-services.yaml (hunk 2 above)
Comment (tonymort) [reply apps/darts-modernisation/darts-stub-services/darts-stub-services.yaml]: removed

**Review by davet1985 — APPROVED** [review]

**Review by willwatters — APPROVED** [review]

**Pull request merged**